src/ApiService/Tests/Integration/AzuriteStorage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.OneFuzz.Service;
+
+using Async = System.Threading.Tasks;
+
+namespace Tests.Integration;
+
+sealed class AzuriteStorage : IStorage {
+    public Uri GetBlobEndpoint(string accountId)
+        => new($"http://127.0.0.1:10000/{accountId}");
+
+    public Uri GetQueueEndpoint(string accountId)
+        => new($"http://127.0.0.1:10001/{accountId}");
+
+    public Uri GetTableEndpoint(string accountId)
+        => new($"http://127.0.0.1:10002/{accountId}");
+
+    public Task<(string?, string?)> GetStorageAccountNameAndKey(string accountId)
+        => Async.Task.FromResult<(string?, string?)>((accountId, "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="));
```

Comment:
move to constant and add a comment with a link to [the doc](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#configure-a-connection-string-for-azurite) to explain where the value comes from.